styles/modules/modals/_about.scss

```diff
@@ -0,0 +1,30 @@
+/* 
```

Comment:
There are some specific styles in there it could work to use in an `_about.scss`

I do not know why the styles did not load. Any ideas?